go-wfs-config.toml

```diff
@@ -1,7 +1,9 @@
 [server]
   host = "localhost"
   port = 9000
-  url = "http://geo.kralidis.ca/go-wfs/"
+  scheme = "http"
```

Comment:
let's prepend these three values with something like "serve_"  or "url_" to help make it clear they go together & are only for building urls.